pkg/kudoctl/cmd/diagnostics/runner_helper.go

```diff
@@ -23,6 +23,21 @@ func diagForInstance(instance string, options *Options, c *kudo.Client, info ver
 		return err
 	}
 
+	deps, err := newDependenciesResources(instance, options, c, s)
+	if err != nil {
+		p.printError(err, DiagDir, "instance")
+		return err
+	}
+
+	for _, dep := range deps {
+		ctx := &processingContext{root: DiagDir, instanceName: dep.instanceObj.Name}
```

Comment:
I'd like for the child instances be placed inside the parent instance folder. currently, `root: DiagDir` results in following diag structure:
```
├── kudo
│   ├── serviceaccountlist.yaml
│   └── servicelist.yaml
├── operator_child
│   ├── child.yaml
│   ├── instance_parent-instance-child
│   │   └── parent-instance-child.yaml
│   └── operatorversion_child-0.0.1
│       └── child-0.0.1.yaml
├── operator_parent
│   ├── instance_parent-instance
│   │   └── parent-instance.yaml
│   ├── operatorversion_parent-0.1.0
│   │   └── parent-0.1.0.yaml
│   └── parent.yaml
├── settings.yaml
└── version.yaml
```
^^ here `operator_child` is on the same level as `operator_parent` while I'd prefer it to be `operator_parent/operator_child`. This nesting will significantly simplify parsing the diagnostics bundle, especially for big operators with complex dependencies.